apps/editing-toolkit/editing-toolkit-plugin/wpcom-universal-themes/index.php

```diff
@@ -352,6 +361,9 @@ function init() {
 	if ( is_core_fse_active() ) {
 		load_core_fse();
 	}
+
+	// always hide the WP core Site Editor.
+	add_action( 'admin_menu', __NAMESPACE__ . '\hide_core_site_editor' );
```

Comment:
Would this be a better fit in the `unload_core_fse` function?

I tested out moving it there as well and the functionality seems the same.

Reply:
Yes, that's what I would expect too.

Reply:
Updated in https://github.com/Automattic/wp-calypso/pull/60499/commits/0c2360d998f1f2ad3299e5dd2fee170af47095e9